cypress/support/index.js

```diff
@@ -0,0 +1,52 @@
+import './commands';
+
+// Initialize console and XHR monitoring
+before(() => {
+  let polyfill;
+  // Polyfills must be in umd format - cypress will coerce (and fail) to load this as a fixture or ES/CJS module
+  cy
+    .log('Polyfill Fetch >>> XHR Fallback')
+    .readFile('node_modules/unfetch/dist/unfetch.umd.js')
```

Comment:
This is pretty much an alternative to `require()`, and we do this so that node doesn't evaluate what it's importing. Instead, we load this file as a string and have the browser evaluate the package.